qcodes/instrument_drivers/stanford_research/SR560.py

```diff
@@ -16,36 +16,41 @@ class VoltageParameter(MultiParameter):
     ``VoltageParameter.get()`` returns ``(voltage_raw, voltage)``
 
     Args:
-        measured_param (Parameter): a gettable parameter returning the
```

Comment:
I like how the doc strings were typed :)

Reply:
We usually remove the types from docstrings once they are added to the signature since they are often wrong and there is no way to check them for correctness in the signatures